jupyterlab/jlpmapp.py

```diff
@@ -57,6 +57,9 @@ def which(command, env=None):
         if command == 'node':
             msg = 'Please install nodejs using conda or the nodejs website'
             raise ValueError(msg)
+        elif command == 'npm':
```

Comment:
Installing `node` provides `npm`, I think this should be part of the block above (`if command in ['node', 'npm']:`)